src/rust/ensogl/src/display/shape/text/text_field/cursor.rs

```diff
@@ -470,9 +481,9 @@ impl Cursors {
     ///
     /// If after this operation some of the cursors occupies the same position, or their selected
     /// area overlap, they are irreversibly merged.
-    pub fn navigate_cursors<Predicate>
-    (&mut self, navigation:&mut CursorNavigation, step:Step, mut predicate:Predicate)
-    where Predicate : FnMut(&Cursor) -> bool {
+    pub fn navigate_cursors<P>
+    (&mut self, navigation:&mut CursorNavigation, step:Step, mut predicate:P)
+    where P : FnMut(&Cursor) -> bool {
```

Comment:
I was reading this code and I was wondering what is `P`. I learned by reading the removed code where it is named `Predicate`. I think the previous name is much better.

Reply:
I just wnader why you're wondering, if the argument name is "Predicate"?